README.md

```diff
@@ -72,9 +72,18 @@ Ensure both SYA (from one of the methods above) and the [API](https://github.com
 present these tests do not run within Docker, therefore, open a new terminal window.
 
 Functional tests:
+We have split our functional tests into two.
+Firstly we have tests for entire journeys through the form:
 
     yarn test:functional
     
+However, in order to get this command to run properly you currently have to remove ``` --grep @functional ``` tag from 
```

Comment:
Is this due to it also running the `@smoke` tests which are not required here? Unless I'm misunderstanding, would this need to be fixed in order to run these functional tests are part of a build?

Reply:
currently tests won't run on pipeline for other reasons, for suppressing tests we have added this.

Reply:
Ok thanks for explaining @vianvi